src/porepy/utils/half_space.py

```diff
@@ -130,10 +130,9 @@ def star_shape_cell_centers(g):
     faces, _, sgn = sps.find(g.cell_faces)
     nodes, _, _ = sps.find(g.face_nodes)
 
-    xn = g.nodes
-    if g.dim == 2:
-        R = cg.project_plane_matrix(xn)
-        xn = np.dot(R, xn)
+    xn = g.nodes.copy()
```

Comment:
As mentioned in #202 this function is not readable to me, and I have issues with in for the python 3.7 update. Would you care to add comments?

Reply:
now the function is documented